ui/src/app/base/types.ts

```diff
@@ -0,0 +1,104 @@
+export type TSFixMe = any; // eslint-disable-line @typescript-eslint/no-explicit-any
+
+export type ErrorState = TSFixMe;
```

Comment:
so I think we just want the errors to have the type TSFixMe, so it's clear that we haven't defined a type for them yet, I don't think we need `ErrorState`.

Reply:
Yeah that's fine - I figured at some point we would define a type for errors in state so we would only need to make the change in one place. But maybe it's better to highlight the fact that it's *not* currently typed wherever it's used.